Clients/src/presentation/components/PluginWidgets/templates/EmptyPageTemplate.tsx

```
@@ -0,0 +1,1047 @@
+        {slot.title}
+      </Typography>
+    )}
+    <Box
+      sx={{
+        p: 2.5,
+        backgroundColor: "#fff",
+        border: "1px solid #d0d5dd",
+        borderRadius: 1,
+        "& img": { maxWidth: "100%" },
+      }}
+      dangerouslySetInnerHTML={{ __html: slot.data.html || "" }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)